lib/omnibus/packagers/msi.rb

```diff
@@ -321,11 +321,11 @@ def package_name
     end
 
     def msi_name
-      "#{project.package_name}-#{project.build_version}-#{project.build_iteration}.msi"
+      "#{project.package_name}-#{project.build_version}-#{project.build_iteration}.#{Config.windows_arch}.msi"
```

Comment:
Looking at how other Windows packages are named ([Ruby](http://rubyinstaller.org/downloads/), [Git](https://git-scm.com/download/win) and [various Chocolatey stuff](https://chocolatey.org/packages?q=64-bit)) I think we should separate iteration and arch with a dash (`-`).